assets/armsnippets.jsonc

```diff
@@ -54,15 +54,6 @@
         ],
         "description": "ARM Template output"
     },
-    "Parameter Value": {

```

Comment:
is there a reason we got rid of this? Is this a snippet to help me add a parameter?

Reply:
It turned out it was a snippet to add a parameter to a template *params* file, which doesn't make sense for arm-template.  However, it does make sense for json, so I'll add it there.